.github/workflows/codespell.yaml

```diff
@@ -0,0 +1,20 @@
+name: codespell
+on:
+  pull_request:
+    branches:
+      - master
+      - enterprise
```

Comment:
Would have been nice if, for reviewers like me who are not proficient in github workflows, you could summarize what this PR would do if committed. E.g., do these lines say that codespell will be run automatically on every PR? Where will the result be visible, and will it be informational or will block the PR if mispellings are found?

What does it actually check - the new text in the PR, or the entire project again and again? If it's the latter, why is interesting to see 100 pre-existing spelling errors again and again every time you send a PR?

I think all of this should be explained in the commit message.

Reply:
@nyh i amended the commit message. does it address your questions now?

Reply:
Yes